chain-impl-mockchain/src/rewards.rs

```diff
@@ -81,6 +84,64 @@ impl TaxType {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
we don't derive Serde stuff usually in chain-libs, favoring a stable implementation defined by the user of this library.

Reply:
This was the quick thing to do at 0400 :)  If it doesn't matter for release, we can fix it up later.